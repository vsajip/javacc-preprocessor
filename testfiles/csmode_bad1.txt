#pragma // We ignore this.
#prag //But we pass this through.
#lines //What about this?

#define 中文 // How cool is this?

#if !中文
This content should not appear.

#else
#

  你好！
  

  Но я не китаец!!!

But ###this!

#oops #undef oops

#endif
This line should appear.
# The following should cause a parse error!
#define _CODELANG = python

