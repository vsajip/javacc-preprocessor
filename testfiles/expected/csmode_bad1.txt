#prag //But we pass this through.


#

  你好！
  

  Но я не китаец!!!

But ###this!

#oops #undef oops

This line should appear.
# The following should cause a parse error!
